libres/lib/enkf/misfit_ensemble.cpp

```diff
@@ -198,10 +187,7 @@ void misfit_ensemble_clear(misfit_ensemble_type *table) {
     table->initialized = false;
 }
 
-void misfit_ensemble_free(misfit_ensemble_type *table) {
-    vector_free(table->ensemble);
-    free(table);
-}
+void misfit_ensemble_free(misfit_ensemble_type *table) { delete table; }
```

Comment:
You still need the `vector_free(table->ensemble)` call - `delete` will just remove the memory set aside for the actual `misfit_ensemble_struct` instance.

Reply:
The C is weak, thanks!